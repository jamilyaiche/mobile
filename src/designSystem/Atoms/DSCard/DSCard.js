import React from 'react'
import { View } from 'react-native'
import Style from '../../../assets/styles/Style'

const DSCard = (props) => {
  return (
        <View style={Style.card}>
            {props.children}
        </View>
  )
}

export default DSCard
