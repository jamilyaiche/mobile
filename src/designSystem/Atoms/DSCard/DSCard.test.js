import React from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import { Text, TouchableOpacity, TextInput as RNTextInput, Pressable } from 'react-native'
import DSCard from './DSCard'
jest.mock('rn-fetch-blob', () => {
    return {
      fs: {
        dirs: {
          DocumentDir: ''
        },
        writeFile: () => Promise.resolve()
      }
    }
  })
  jest.mock('react-native-x2-carousel', () => {
    return {
      fs: {
        dirs: {
          DocumentDir: ''
        },
        writeFile: () => Promise.resolve()
      }
    }
  })
describe('DSCard tests', () => {
  it('test default DSCard snapshot', () => {
    const wrapper = renderer.create(<DSCard></DSCard>)
      .toJSON()
    expect(wrapper).toMatchSnapshot()
  })
  it('test default DSCard with text snapshot', () => {
    const wrapper = renderer.create(<DSCard><Text>Test</Text></DSCard>)
      .toJSON()
    expect(wrapper).toMatchSnapshot()
  })
 
})
