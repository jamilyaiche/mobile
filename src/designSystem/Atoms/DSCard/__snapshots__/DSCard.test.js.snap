// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DSCard tests test default DSCard snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(255, 255, 255, 1)",
      "borderRadius": 10,
      "elevation": 4,
      "height": "100%",
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.2,
      "shadowRadius": 5,
      "width": "100%",
    }
  }
/>
`;

exports[`DSCard tests test default DSCard with text snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgba(255, 255, 255, 1)",
      "borderRadius": 10,
      "elevation": 4,
      "height": "100%",
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.2,
      "shadowRadius": 5,
      "width": "100%",
    }
  }
>
  <Text>
    Test
  </Text>
</View>
`;
